jdaviz/configs/imviz/plugins/compass/compass.py

```diff
@@ -3,12 +3,24 @@
 from jdaviz.core.events import AddDataMessage, RemoveDataMessage
 from jdaviz.core.registries import tray_registry
 from jdaviz.core.template_mixin import PluginTemplateMixin, ViewerSelectMixin
+from jdaviz.core.user_api import PluginUserApi
 
 __all__ = ['Compass']
 
 
 @tray_registry('imviz-compass', label="Imviz Compass")

```

Comment:
do we want to rename this to just "Compass"?

Reply:
Hmm... Come to think about it, maybe. Especially if we're moving towards more dynamic configurations, perhaps we should be proactive and begin stripping out the config-specific labels?

Reply:
I would be in favor of that.  We may not ever put this plugin in a different tool, but if we do, I can't imagine we'd have separate plugins entirely... we would want to generalize this one.  Any opposition @pllim?  Note that this affects the dictionary key to access the API _and_ the text in the plugin tray expansion menus.

Reply:
I have no idea if this will work as-is on Cubeviz, Mosviz, and Specviz2d image viewers. Probably not.

Reply:
No, I wouldn't expect it to out-of-the-box without some generalizing.  Is that the intention of "Imviz" in the plugin name?  Is there some other way we can document that internally without it being in the user-facing title?

Reply:
Naming is hard... If you want to remove "Imviz", sure, go ahead...

Reply:
Remove Imviz you say? 👀 😆

Reply:
Re: https://github.com/spacetelescope/jdaviz/pull/1643#discussion_r969998936

I think that is a good compromise. I guess if you see a plugin in Imviz GUI already, it is pretty obvious to the user.